### PR TITLE
BXMSPROD-1208 prod profile

### PR DESCRIPTION
**Pull request by Ginxo**
https://issues.redhat.com/browse/BXMSPROD-1208

https://github.com/kiegroup/optaplanner/pull/1165
https://github.com/kiegroup/kogito-examples/pull/568
https://github.com/kiegroup/kogito-runtimes/pull/1078
https://github.com/kiegroup/kogito-pipelines/pull/117

### CODE REVIEW

**Review by radtriste — APPROVED** [review]

File: Jenkinsfile.prod.nightly [hunk 1]
```diff
@@ -41,7 +41,7 @@ pipeline {
                     pipelineHelper.retry(
                     {
                         def SETTINGS_XML_ID = "5d9884a1-178a-4d67-a3ac-9735d2df2cef"
-                        def projectCollection = ["kogito-runtimes", "optaplanner", "kogito-apps", "kogito-examples"]
+                        def projectCollection = ["kogito-runtimes", "optaplanner", "kogito-examples"]
```
Comment (mareknovotny) [inline Jenkinsfile.prod.nightly:44]: i think we still need kogito-apps

**Review by cristianonicolai — APPROVED** [review]

File: Jenkinsfile.prod.nightly (hunk 1 above)
Comment (Ginxo) [reply Jenkinsfile.prod.nightly:44]: @mareknovotny do we still waiting for @tkobayas @cristianonicolai @evacchi confirmation?

Comment (mareknovotny) [reply Jenkinsfile.prod.nightly:44]: i think @radtriste  confirmed it is not needed on the jira. So we can keep your change here

**Review by mareknovotny — APPROVED** [review]
lgtm

**Review by evacchi — APPROVED** [review]

**Pull request merged**